assets/sass/components/btn.scss

```diff
@@ -172,4 +185,26 @@ $btnConf: (
             height: 36px;
         }
     }
-}
\ No newline at end of file
+}
+
+/* =========================================================================
+   Button Group
+   ========================================================================= */
+/* Group together a set of buttons with consistent spacing */
+
+.o-button-group {
```

Comment:
Not sure we have something like this already, but this introduces the concept of a group of buttons to aid with spacing them consistently in a set. The `.o-` prefix is for "Objects" i.e. lower level structural components like buttons and icons. I tend to quite closely follow Harry Roberts naming conventions for this kind of thing, see: https://csswizardry.com/2015/03/more-transparent-ui-code-with-namespaces/ but can rename this prefix if there's existing terminology that better fits (e.g. is this more appropriately "scaffolding"?)

Reply:
👍  sensible choice. There's probably a juicy refactoring task here to rename some existing classes in this way (eg. see the hackery towards the end of `layout.scss`)